### ISSUE TITLE
Acy GUI: catastrophic accessibility

### ISSUE THREAD

**Issue by hytsch** [acyba/acymailing#68]
Acy GUI with catastrophic accessibility: Contrasts at the lowest limit of recognizability! In addition, the dark mode in Joomla, which would be a lifesaver for people with normal eyesight, simply doesn't work at all.
Best regards

![Image](https://github.com/user-attachments/assets/d122a3d1-bde4-4780-a3ce-730fb6566eba)

**Comment by roumilb** [comment]
Thanks for reporting this, this is not the normal UI it's a display issue.
What device are you using?

Also, do you have other screenshots to share or it's only the datepicker?
